src/current/cockroachcloud/upgrade-policy.md

```diff
@@ -9,48 +9,67 @@ This page describes the support and upgrade policy for clusters deployed in Cock
 
 ## CockroachDB Cloud Support Policy
 
-[Major versions]({% link releases/index.md %}) of CockroachDB are labeled either [Regular releases]({% link releases/index.md %}#major-releases) or [Innovation releases]({% link releases/index.md %}).
+CockroachDB {{ site.data.products.cloud }} clusters are intended to always run a supported, recent major version of CockroachDB. Clusters receive regular patch updates to ensure they have the latest stability and security updates. Different {{ site.data.products.cloud }} plans offer different options for version selection and upgrade windows, as described on this page.
+
+A cluster running an unsupported CockroachDB version is not eligible for Cockroach Labs’ [availability SLA](https://www.cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/).
+
+### Version and upgrade support by cloud plan
+
+[Major versions]({% link releases/index.md %}) of CockroachDB are labeled either as [Regular releases]({% link releases/index.md %}#major-releases) or [Innovation releases]({% link releases/index.md %}).
 
 - **Regular releases** are supported for 12 months from their initial production release date.
 - **Innovation releases** are supported for 6 months from their initial production release date.
 
-For each release type, the end date of this period is called End of Support (EOS).
-
-A cluster running an unsupported CockroachDB version is not eligible for Cockroach Labs’ [availability SLA](https://www.cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/).
+CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} clusters only support **Regular releases**. This ensures that these clusters are using versions of CockroachDB that receive a full year of support with routine patch updates. All CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} clusters are automatically upgraded to the next major Regular release upon its GA release.
```

Comment:
There is a nuance here. All Basic clusters are automatically upgraded. If you have Standard cluster and have enabled [manual upgrade](https://www.cockroachlabs.com/docs/cockroachcloud/cluster-management#set-major-version-upgrades-to-automatic-or-manual), the cluster will not be automatically upgraded. The customer will have to take the action themselves. In this case, if the customer is running an EOS version (For standard, Regular release is EOS 12 months after the date of production release), they will be automatically upgraded to the next regular release.
If the customer has no manual upgrade set on Standard, they will be automatically upgraded